addons/dragging/functions/fnc_dropObject_carry.sqf

```diff
@@ -31,6 +31,11 @@ private _inBuilding = [_unit] call FUNC(isObjectOnObject);
 // release object
 detach _target;
 
+// Set to default value
+private _parentConfig = [_target] call CBA_fnc_getObjectConfig;
+private _defaultPos = getArray(_parentConfig >> QGVAR(carryPosition));
+_target setVariable [QGVAR(carryPosition), _defaultPos, true];
```

Comment:
This breaks `ace_dragging_fnc_setCarryable` because the object has no config, so it stores `[]`